languages/wikipron/src/scrape.py

```diff
@@ -66,99 +30,108 @@ def _call_scrape(lang, config, filename_suffix):
                 logging.info(
                     'Exception detected while scraping: "%s", "%s". Restarting.',
                     lang,
-                    filename_suffix,
+                    tsv_path,
                 )
                 # Pauses execution for 10 min.
                 time.sleep(600)
 
 
+def build_config_and_filter_scrape_results(
+    config_settings, wiki_name,
+    dialect_extension=''
+):
+    phonemic_config = wikipron.Config(
+        **config_settings,
+    )
+    phonemic_path = (
+        f"../tsv/{config_settings['key']}{dialect_extension}_phonemic.tsv"
+    )
+    phonemic_count = _call_scrape(
+        config_settings["key"], phonemic_config, phonemic_path
+    )
+
+    phonetic_config = wikipron.Config(
+        phonetic=True,
+        **config_settings,
+    )
+    phonetic_path = (
+        f"../tsv/{config_settings['key']}{dialect_extension}_phonetic.tsv"
+    )
+    # Skips phonetic if phonemic failed to complete scrape.
+    if phonemic_count is not None:
+        phonetic_count = _call_scrape(
+            config_settings["key"], phonetic_config, phonetic_path
+        )
+    # Remove files for languages that failed to be scraped
+    # within set amount of retries.
+    if phonemic_count is None or phonetic_count is None:
+        logging.info(
+            'Failed to scrape "%s", moving on to next language. %s',
+            wiki_name,
+            {config_settings["key"]: config_settings}
+        )
+        if os.path.exists(phonemic_path):
+            os.remove(phonemic_path)
+        if os.path.exists(phonetic_path):
+            os.remove(phonetic_path)
+        return
+    # Remove files for languages that returned nothing
+    elif phonemic_count == 0 and phonetic_count == 0:
+        logging.info(
+            '"%s", returned no entries in phonemic and phonetic settings.',
+            wiki_name,
+        )
+        os.remove(phonemic_path)
+        os.remove(phonetic_path)
+        return
+    # Removes TSV files with less than 100 entries.
+    if phonemic_count < 100:
+        logging.info(
+            '"%s", has less than 100 entries in phonemic transcription.',
```

Comment:
what's with the comma in the log statement? This is going to say:

`Amharic, has less than 100 entries...`

Is that intended?